src/components/counter.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {countItems,addItems} from '../actions/actions.js';
+import store from '../store/store.js';
+
+
+const Counter = React.createClass({
+	render:function(){
+		return (
+		<div>
+		<h1> ADDED ITEMS : </h1>
+			<button onClick={store.dispatch.bind(this,countItems(1))}>Shirts</button>
```

Comment:
Your dispatch actions are all being called with the same argument of 1. Why is that?